daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md

```diff
@@ -149,6 +149,26 @@ To find out more details on the purpose of any of these metadata properties, ple
 
 > Note that all times are populated by the server and are not adjusted for clock skews.
 
+## Sending and receiving multiple messages
+
+Azure Service Bus natively supports sending and receiving multiple messages in a single operation. 
+
+### Configuring bulk publish
+
+To set the metadata for bulk publish operation, set the query parameters on the HTTP request or the gRPC metadata as documented [here]({{< ref pubsub_api >}})
+
+| Metadata | Default |
+|----------|---------|
+| `metadata.maxBulkPubBytes` | `131072` (128 KiB) |
```

Comment:
These fields needs to be added to the component spec example at the start as well as into the table of values. Putting them here does not make them discoverable. The table is the source of truth for the spec.

Reply:
I see the same issue on needing the table in the Kafka Docs PR

Reply:
> So this is not a part of the component spec. A link will be added here with https://github.com/dapr/docs/pull/3077 that will show how to use this configuration, it will be a part of the response from /dapr/subscribe.

Same as other.